docs/quickstart-packet.md

```diff
@@ -0,0 +1,269 @@
+  `Installing KubeOne` section of [the
+  README](https://github.com/kubermatic/kubeone/blob/master/README.md),
+* `terraform` installed. The binaries for `terraform` can be found on the
+  [Terraform website](https://www.terraform.io/downloads.html)
+
+## Setting Up Credentials
+
+In order for Terraform to successfully create the infrastructure and for KubeOne
+to install Kubernetes and create worker nodes you need an API Access Token. You
+can refer to [the official
+documentation](https://support.packet.com/kb/articles/api-integrations) for
+guidelines for generating the token.
+
+Once you have the API access token you need to set the `PACKET_AUTH_TOKEN` and
+`PACKET_PROJECT_ID` environment variables:
+
+```bash
+export PACKET_AUTH_TOKEN=<api key>
+export PACKET_PROJECT_ID=<project UUID>
+```
+
+**Note:** The API access token is deployed to the cluster to be used by
+`machine-controller` for creating worker nodes.
+
+## Creating Infrastructure
+
+KubeOne is based on the Bring-Your-Own-Infra approach, which means that you have
+to provide machines and needed resources yourself. To make this task easier we
+are providing Terraform scripts that you can use to get started. You're free to
+use your own scripts or any other preferred approach.
+
+The Terraform scripts for Packet are located in the
+[`./examples/terraform/packet`](https://github.com/kubermatic/kubeone/tree/master/examples/terraform/packet)
+directory.
+
+**Note:** KubeOne comes with Terraform integration that is capable of reading
+information about the infrastructure from Terraform output. If you decide not to
+use our Terraform scripts but want to use Terraform integration, make sure
+variable names in the output match variable names used by KubeOne.
+Alternatively, if you decide not to use Terraform, you can provide needed
+information about the infrastructure manually in the KubeOne configuration file.
+
+First, we need to switch to the directory with Terraform scripts:
+
+```bash
+cd examples/terraform/packet
+```
+
+Before we can use Terraform to create the infrastructure for us, Terraform needs
+to download the Packet provider and setup it's environment. This is done by
+running the `init` command:
+
+```bash
+terraform init
+```
+
+**Note:** You need to run this command only the first time before using scripts.
+
+You may want to configure the provisioning process by setting variables defining
+the cluster name, device type, facility and similar. The easiest way is to
+create the `terraform.tfvars` file and store variables there. This file is
+automatically read by Terraform.
+
+```bash
+nano terraform.tfvars
+```
+
+For the list of available settings along with their names please see the
+[`variables.tf`](https://github.com/kubermatic/kubeone/blob/master/examples/terraform/packet/variables.tf)
+file. You should consider setting:
+
+* `cluster_name` (required) - prefix for cloud resources
+* `project_id` (required) - packet project UUID
```

Comment:
Oh, you have added the project ID here. Not sure what is the most common way, this or an environment variable.

Reply:
packet provider (in docs) https://www.terraform.io/docs/providers/packet/index.html#argument-reference doesn't support passing project_id as env